Scripts/Mac/RunFunctionalTests.sh

```diff
@@ -7,9 +7,8 @@ if [ -z $CONFIGURATION ]; then
   CONFIGURATION=Debug
 fi
 
-sudo mkdir /GVFS.FT
-sudo chown $USER /GVFS.FT
-
+sudo mkdir ~/GVFS.FT
+sudo chown $USER ~/GVFS.FT
```

Comment:
Is this line still needed (now that we're creating the directory in HOME)?

Reply:
Course not :-)